HealthSDK/GiniHealthSDK/Tests/GiniHealthSDKTests/GiniHealthTests.swift

```diff
@@ -560,4 +560,86 @@ final class GiniHealthTests: XCTestCase {
         XCTAssertEqual(clientConfiguration.communicationTone, expectedDefaultComunicationTone)
         XCTAssertEqual(clientConfiguration.ingredientBrandType, expectedDefaultBrandType)
     }
+
+    // MARK: - Delete Batch Of Documents Tests
```

Comment:
[Clarification]: Can we extract some tests in new file, so far we have >=500 lines?